capsules/extra/src/ieee802154/mac.rs

```diff
@@ -146,6 +146,19 @@ impl<'a, R: radio::Radio<'a>> Mac<'a> for AwakeMac<'a, R> {
         full_mac_frame: &'static mut [u8],
         frame_len: usize,
     ) -> Result<(), (ErrorCode, &'static mut [u8])> {
+        // We must add the PSDU_OFFSET required for the radio
```

Comment:
To check that I understand this PR:

1) You have changes in `driver.rs` that properly account for the three-byte header when receiving packets.

2) You have removed the two-byte PDSU header from received packets. This changed `framer.rs` in the `client.receive()` calls.

2) Unrelated to that first two items, you have changed transmitted buffers to only be the data, not a two-byte header and then data. That required changes to `framer.rs` and `mac.rs` but apparently no other files.

Is that right? I think it seems in agreement with the commit history.